### DISCUSSION TITLE
How to mute default info logs

### DISCUSSION THREAD

**Discussion by ayhoung** [webdriverio/webdriverio#7114] [Q&A] [answered]
Hi all, I'm using webdriver to crawl a specific page, which looks like the following:

```
import { remote } from 'webdriverio';

(async () => {
    const data = [];
    const browser = await remote({
        capabilities: {
            browserName: 'chrome'
        }
    });

    await browser.url('https://www.example.com);
    const posts = browser.$$('.class');

    try {
        Promise.each(posts, async (item) => {
            try {
                const titleElement = await item.$('.titleClass');
                ...
               console.log(titleElement.getText());
            } catch (err) {
                // console.log(err);
            }
        });
    } catch (err) {
        console.log(err);
    }
})();
```

How do I mute/silence the default logs and only display the logs I've logged myself, in this case `console.log(titleElement.getText())`

By default, webdriver prints out a lot of logs such as 
`2021-07-11T10:32:49.981Z INFO devtools: COMMAND findElementsFromElement("ELEMENT-8", "css selector", ".class")`

**Comment by christian-bromann** [comment, marked as answer]
Please see the [docs](https://webdriver.io/docs/options#loglevel). If you set `logLevel: "error"` you should mostly see your log information.

**Reply by ayhoung** [reply]
thanks a lot! :)
